DigitalLearningSolutions.Data.Tests/TestHelpers/LearningLogItemsTestHelper.cs

```diff
@@ -0,0 +1,78 @@
+﻿namespace DigitalLearningSolutions.Data.Tests.TestHelpers
+{
+    using System;
+    using Dapper;
+    using Microsoft.Data.SqlClient;
+
+    public class LearningLogItemsTestHelper
+    {
+        private readonly SqlConnection connection;
+
+        public LearningLogItemsTestHelper(SqlConnection connection)
+        {
+            this.connection = connection;
+        }
+
+        public LearningLogItemDetails? SelectLearningLogItemWithResourceLink(string resourceLink)
+        {
+            return connection.QuerySingleOrDefault<LearningLogItemDetails>(
+                @"SELECT
+                        LoggedDate AS AddedDate,
+                        LoggedByID AS DelegateId,
+                        Activity AS ResourceName,
+                        ExternalUri AS ResourceLink,
+                        LinkedCompetencyLearningResourceID AS CompetencyLearningResourceId
+                    FROM LearningLogItems
+                    WHERE ExternalUri = @resourceLink",
+                new { resourceLink }
+            );
+        }
+
+        public CandidateAssessmentLearningLogItem? SelectCandidateAssessmentLearningLogItem()
+        {
+            return connection.QuerySingleOrDefault<CandidateAssessmentLearningLogItem>(
+                @"SELECT
+                        ID,
+                        CandidateAssessmentID,
+                        LearningLogItemID
+                    FROM CandidateAssessmentLearningLogItems"
+            );
+        }
+
+        public LearningLogItemCompetency? SelectLearningLogItemCompetency()
+        {
+            return connection.QuerySingleOrDefault<LearningLogItemCompetency>(
+                @"SELECT
+                        ID,
+                        LearningLogItemID,
+                        CompetencyID,
+                        AssociatedDate
+                    FROM LearningLogItemCompetencies"
+            );
+        }
+
+        public class LearningLogItemDetails

```

Comment:
We are going to be using LearningLogItems in a bunch of other tickets. Do you reckon its worth moving this class into its own file under Models, to reduce the number of potentially redundant classes when everything gets merged?

Reply:
Added this with columns names as per the database. We'll probably want to rename columns as we actually use this properly later.